downstream/modules/platform/con-controller-cleanup-expired-sessions.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-controller-cleanup-expired-sessions"]
+
+= Cleanup Expired Sessions
+
+To remove expired sessions, click on the launch image:rightrocket.png[Launch,15,15] icon beside *Cleanup Expired Sessions*.
+
+You can review or set a schedule for cleaning up expired sessions by performing the same procedure described for activity stream management jobs. 
+For more information, see [Scheduling].
```

Comment:
Is this for the section "proc-controller-scheduling-deletion"?

see xref:proc-controller-scheduling-deletion[Scheduling deletion].